src/sage/features/sagemath.py

```diff
@@ -560,7 +560,7 @@ def __init__(self):
         """
         JoinFeature.__init__(self, 'sage.libs.giac',
                              [PythonModule('sage.libs.giac.giac')],
-                             spkg='sagemath_giac', type='standard')
+                             spkg='sagemath_giac', type='optional')
```

Comment:
Should this feature not better now use the new sagemath-giac package to test it? `sage.libs.giac.giac` is always available.

Alternatively, if we no longer need the giac tests (because all tests are now in the new giac repo), then please remove it.

Reply:
This still works because the `PythonModule` test tries to import the module and it will fail if sagemath-giac is not installed:

```
sage: from sage.features.sagemath import sage__libs__giac
sage: sage__libs__giac().is_present()
FeatureTestResult('sage.libs.giac.giac', False)
```

There are indeed some integral tests in sagelib that have `# needs sage.libs.giac`. I did some work a month or two ago to carefully separate them from the `# needs giac` tests.